pkg/cas/resolver/resolver.go

```diff
@@ -106,11 +106,6 @@ func (h *Resolver) Resolve(webCASURL *url.URL, cidWithPossibleHint string, //nol
 			return h.resolveCIDWithHint(cidWithPossibleHintParts)
 		}
 
-		if h.ipfsReader != nil {
```

Comment:
With this PR, when writing to local CAS we now write to IPFS as well. However, we always only read from local CAS (we do not use IPFS as a fallback). I think it would be weird if we kept this fallback in only this one specific case. I feel like we should either always use IPFS as a fallback or never (when it's configured, of course). Let me know if you see any reason why we should only do the fallback here... or if you think we should always be using IPFS as a fallback (again, if configured)